nixos/doc/manual/installation/installing.chapter.md

```diff
@@ -515,15 +523,14 @@ Use the following commands:
     menu. This allows you to easily roll back to a previous
     configuration if something goes wrong.
 
-    You should log in and change the `root` password with `passwd`.
+    Use your declared user account to log in. If you didn’t
+    declare one, you should still be able to log in using
+    the `root` user.
```

Comment:
Use one line per sentence, this is easier on the diffs (already to be seen in review suggestions). No need to follow the surrounding format. 

```suggestion
    Use your declared user account to log in.
    If you didn’t declare one, you should still be able to log in using the `root` user.
```